<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MicrologicIDE</class>
 <widget class="QMainWindow" name="MicrologicIDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MicrologicIDE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="debugText">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>72</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::LineWrapMode::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveAsFileAction"/>
    <addaction name="separator"/>
    <addaction name="exitNotepadAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="separator"/>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="deleteAction"/>
    <addaction name="separator"/>
    <addaction name="selectAllAction"/>
    <addaction name="dateAction"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="lightModeAction"/>
    <addaction name="darkModeAction"/>
    <addaction name="safeModeAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="getHelpAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_F5">
    <property name="title">
     <string>运行(R)</string>
    </property>
    <addaction name="runAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_V"/>
   <addaction name="menu_F5"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsFileAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/saveAs.png</normaloff>:/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)</string>
   </property>
  </action>
  <action name="exitNotepadAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除(L)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="selectAllAction">
   <property name="text">
    <string>全选(A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="dateAction">
   <property name="text">
    <string>时间/日期(D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="getHelpAction">
   <property name="text">
    <string>查看帮助(H)</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="runAction">
   <property name="text">
    <string>运行(F5)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="darkModeAction">
   <property name="text">
    <string>深色模式</string>
   </property>
  </action>
  <action name="lightModeAction">
   <property name="text">
    <string>浅色模式</string>
   </property>
  </action>
  <action name="safeModeAction">
   <property name="text">
    <string>安全模式-未开启</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
